### ISSUE TITLE
Unhandled Promise rejection for non error object

### ISSUE THREAD

**Issue by WaeCo** [mreuvers/typescript-logging#37]
Passing a non error like object to a logger function will throw an unhanded promise rejection.

Example:
```
const noErr = { foo: "bar" }
logger.error("message", noErr);
//Error: unhandled promise rejection in ...
```

This happens to me sometimes when other libraries throw non error like objects and I have a generic catch block.

It can be mitigated by adding a catch  handler in
https://github.com/mreuvers/typescript-logging/blob/13afee10c5509ce0ab8f8bf0754002681307e899/src/logging/utils/MessageUtils.ts#L196
and
https://github.com/mreuvers/typescript-logging/blob/13afee10c5509ce0ab8f8bf0754002681307e899/src/logging/log/standard/AbstractLogger.ts#L340

But I am unsure what the output should be.
Maybe handle the error like a data object?

**Comment by mreuvers** [comment]
That makes sense I agree. I will fix it and release a patch version later this week.

**Comment by mreuvers** [comment]
This should be fixed now, published it to npm as well, as 0.6.2 release. Feel free to test it. I will close the issue in a few days from now, assuming it is fine by then.
